.env.template

```diff
@@ -42,31 +42,31 @@ ETH_RPC=
 # Otherwise the pathfinding service won't start.
 # Please make sure to read the disclaimer and only uncomment once you have read and accepted the disclaimer.
 # Please read: https://github.com/raiden-network/raiden-services/blob/f4bcb9c289e093754204fe18684e4b57558ea29b/src/pathfinding_service/constants.py#L34
-#PFS_ACCEPT_DISCLAIMER=TRUE
+# PFS_ACCEPT_DISCLAIMER=TRUE
 
 # The monitoring service disclaimer needs to be accepted by setting ACCEPT_SERVICES_DISCLAIMER as written below.
 # Otherwise the monitoring service won't start.
 # Please make sure to read the disclaimer and only uncomment once you have read and accepted the disclaimer.
 # Please read: https://github.com/raiden-network/raiden-services/blob/f4bcb9c289e093754204fe18684e4b57558ea29b/src/monitoring_service/constants.py#L22
-#MS_ACCEPT_DISCLAIMER=TRUE
+# MS_ACCEPT_DISCLAIMER=TRUE
 
 ## Specify the chain name/id of the Ethereum network to run Raiden on.
 # Available networks:
-# - "mainnet"
-# - "ropsten"
-# - "rinkeby"
-# - "goerli"
-# - "kovan"
-# - "<CHAIN_ID>" use the given numeric chain id directly
+# - mainnet
+# - ropsten
+# - rinkeby
+# - goerli
+# - kovan
+# - <CHAIN_ID> use the given numeric chain id directly
 ## For example on the goerli testnet this would become: CHAIN_ID=goerli
 CHAIN_ID=
 
 ## PFS fee, can be 0
 PFS_SERVICE_FEE=100
 
 ## PFS info settings, both are shown in the client during startup
-# PFS_OPERATOR="<OPERATOR_NAME>"
-# PFS_INFO_MESSAGE="<PFS_INFO_MESSAGE>"
+# PFS_OPERATOR=<OPERATOR_NAME>
```

Comment:
This should fix the operator and message fields in https://pfs.rsb01.test.raiden.network/api/v1/info